README.md

```diff
@@ -1,20 +1,29 @@
 <!-- 
 Add a project state badge
-
-See <https://github.com/BCDevExchange/Our-Project-Docs/blob/master/discussion/projectstates.md> 
-If you have bcgovr installed and you use RStudio, click the 'Insert BCDevex Badge' Addin.
--->
-[![Lifecycle:Maturing](https://img.shields.io/badge/Lifecycle-Maturing-007EC6)](Redirect-URL)
+<!-- badges: start -->
+[![Lifecycle:Experimental](https://img.shields.io/badge/Lifecycle-Experimental-339999)](https://github.com/bcgov/repomountie/blob/master/doc/lifecycle-badges.md)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/license/apache-2-0/)
+<!-- badges: end -->
 
 
 BC community Socio-economic status index
 ============================
 
-
 BC community socio-economic status (SES) index evaluates individuals or groups' relative social and economic standing within a BC.
 
 The SES index is often used in research, public policy, and social programs to understand disparities in outcomes such as health, education, and employment, and to develop interventions aimed at reducing inequality and promoting social mobility.
```

Comment:
I would add a line along the lines of...

"This repo is utilized to clean externally available sources of data such that it can be ingested for further use within a secure research environment."

Reply:
I have added it.